File: ProAgil-App/src/app/_models/Palestrante.ts
import { RedeSocial} from './RedeSocial';
import { Evento} from './Evento';
export interface Palestrante {
        id: number;
        name: string;
        miniCurriculo: string;
        imagemUrl: string;
        telefone: string ;
        email: string ;
        redesSociais: RedeSocial[];
        palestrantesEvento: Evento[];

}

File: ProAgil.Domain/Palestrante.cs
using System.Collections.Generic;

namespace ProAgil.Domain
{
    public class Palestrante
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string MiniCurriculo { get; set; }   
        public string ImagemUrl { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }

        public List<RedeSocial> RedesSociais { get; set; }
        public List<PalestranteEvento> PalestrantesEvento { get; set; }
    }
}